src/test/java/redis/clients/jedis/tests/commands/SortedSetCommandsTest.java

```diff
@@ -1298,6 +1298,40 @@ public void zunionstoreParams() {
     assertEquals(bexpected, jedis.zrangeWithScores(SafeEncoder.encode("dst"), 0, 100));
   }
 
+  @Test
+  public void zinter() {
+    jedis.zadd("foo", 1, "a");
+    jedis.zadd("foo", 2, "b");
+    jedis.zadd("bar", 2, "a");
+
+    ZParams params = new ZParams();
+    params.weights(2, 2.5);
+    params.aggregate(ZParams.Aggregate.SUM);
+    Set<String> expected = new LinkedHashSet<>();
+    expected.add("a");
+    assertEquals(expected, jedis.zinter(params, "foo", "bar"));
```

Comment:
You can use `Collections.singleton()` in here and following places.
```suggestion
    assertEquals(Collections.singleton("a"), jedis.zinter(params, "foo", "bar"));
```